bedrock/base/templates/base-resp.html

```diff
@@ -167,13 +167,36 @@
     </div><!-- close #outer-wrapper -->
 
     <!--[if IE 9]>
-      <script src="{{ static('js/libs/matchMedia.js') }}"></script>
+      {% javascript 'matchmedia' %}
     <![endif]-->
 
+    {#
+      The logic mess below will be simplified when all pages are refactored
+      to behave well in old IE without page-specific JS.
+    #}
+    {% if force_modern_js != True %}
```

Comment:
Does this conditional become somewhat redundant, seeing as we're keeping the `no-js` class set for IE < 9 in `site.js`? For example, the homepage has `'force_modern_js': True` in the view, yet it still gets the no-JS styling. Could this potentially lead to breakages introduced by the page JS expecting the `js` class to have been applied?

I think as long as pages look OK with `no-js` (and most do?), then would we even need this? This could simplify a fair bit of cruft in the templates/views, and also puts us further toward dropping support for realz.

Thoughts?